docs/swarm/swarm-mode.md

```diff
@@ -0,0 +1,176 @@
+
+    >**Note**: Docker recommends you use the `--listen-addr` flag to
+    specify precisely the IP address and port where the manager should
+    listen.
+
+* Creates a root CA for the swarm and implements the default security policy.
+   More in depth documentation about about mutual TLS Authentication is forthcoming.
+
+* Implements a raft consensus model for managing the swarm state and the desired
+state for services.
+
+More topics are forthcoming to help you plan for deploying services into
+a production swarm.
+
+### Configure node listen address
+
+When you initialize a swarm the Engine automatically configures a listen address
+for manager nodes based upon the available network interfaces.
+
+The manager node uses the listens address for cluster management
+communications, such as requests for new nodes to join the swarm. The
+other nodes on the swarm must be able to access the manager node on the
+IP address and port you specify for the listen address.
+
+Especially when there are multiple active network interfaces, Docker
+recommends you explicitly define the listen address when you initialize
+a swarm:
+
+  ```
+  docker swarm init --listen-addr <MANAGER-IP>:<PORT>
+  ```
+
+### Set acceptance policy
+
+When a new node attempts to join the swarm, the swarm manager enforces
+an acceptance policy. You can configure the swarm to automatically
+accept new nodes into the swarm as follows:
+
+* **Do not automatically accept nodes**: From a manager node, you must approve
+requests to join the swarm from all manager nodes and all worker nodes.
+
+    ```
+    docker swarm init --auto-accept none
+    ```
+
+    This is the most secure option. However, it requires intervention to
+    accept every new node into the swarm.
+
+* **Automatically accept workers**: The swarm manager automatically
+approves any requests to join the swarm from worker nodes. From a
+manager node, you must approve all manager nodes requesting to join the
+swarm.
+
+    ```
+    docker swarm init --auto-accept worker
+    ```
+
+* **Automatically accept managers**: The swarm manager automatically
+approves any requests to join the swarm from manager nodes. From a
+manager node, you must approve all worker nodes requesting to join the
+swarm.
+
+     ```
+     docker swarm init --auto-accept manager
+     ```
+
+     This option is not secure and not recommended for production environments.
+
+* **Automatically accept workers and managers**: The swarm manager
+automatically approves requests from all nodes to join the swarm.
+
+    ```
+    docker swarm init --auto-accept manager --auto-accept worker
+    ```
+
+    This option is the least secure. Only use this configuration in non-production
+    environments.
+
+
+### Set a shared secret
+
+You can set a text string secret for the swarm. Each
+subsequent request to join the cluster must include the secret.
+
+```
+docker swarm init --secret my$warm$ecret
```

Comment:
Don't use dollar signs in the example.  `$warm` and `$ecret` will be probably be expanded by the shell into an empty string.  Just use letters and numbers.